src/onnx/parse_batchnorm.cpp

```diff
@@ -47,6 +47,13 @@ struct parse_batchnorm : op_parser<parse_batchnorm>
         auto x_lens = args[0]->get_shape().lens();
         auto x_type = args[0]->get_shape().type();
 
+        if(std::any_of(args.cbegin() + 1, args.cend(), [](auto a) {
+               return a->get_shape().lens().size() != 1;
+           }))
+        {
+            MIGRAPHX_THROW("PARSE_BATCHNORM: argument scale, bias, mean, or var rank != 1");
+        }
```

Comment:
isn't it guaranteed to be rank 1 by onnx spec ?

Reply:
I might've misunderstood your previous comment, let me know what you meant.

Reply:
as given here https://github.com/onnx/onnx/blob/main/docs/Operators.md#BatchNormalization 

```
scale (differentiable) : T1
Scale tensor of shape (C).
B (differentiable) : T1
Bias tensor of shape (C).
input_mean (differentiable) : T2
running (training) or estimated (testing) mean tensor of shape (C).
input_var (differentiable) : T2
running (training) or estimated (testing) variance tensor of shape (C).

```

Isn't it dictated by onnx spec that rank would be guaranteed to be 1 ?